src/cljs/org/broadinstitute/firecloud_ui/common/table_utils.cljs

```diff
@@ -24,7 +24,7 @@
 (react/defc Paginator
   {:render
    (fn [{:keys [props]}]
-     (let [{:keys [pagination-params num-visible-rows]} props
+     (let [{:keys [pagination-params num-visible-rows num-total-rows]} props
```

Comment:
Minor style nit: I tend to prefer foo-count over num-foo. In this case, you're being consistent with existing style, which is good, so I'm not suggesting a change.